### ISSUE TITLE
psycopg2 installing error

### ISSUE THREAD

**Issue by Devkalion** [carrotquest/django-clickhouse#15]
Could you use in requirements `psycopg2-binary` instead of `psycopg2` because `psycopg2` requires pg_config which is not so necessary.

Install error:
```
Error: pg_config executable not found.
    
    pg_config is required to build psycopg2 from source.  Please add the directory
    containing pg_config to the $PATH or specify the full executable path with the
    option:
    
        python setup.py build_ext --pg-config /path/to/pg_config build ...
    
    or with the pg_config option in 'setup.cfg'.
    
    If you prefer to avoid building psycopg2 from source, please install the PyPI
    'psycopg2-binary' package instead.
    
    For further information please check the 'doc/src/install.rst' file (also at
    <https://www.psycopg.org/docs/install.html>).
```

**Comment by M1ha-Shvn** [comment]
Hi. 
Thanks for reply. 
To be the truth, I'm not sure if psycopg2 should be still mentioned in requirements as other databases are supported as well. So, I think, I'll remove it from non-test requirenents. You can make a PR in order to make it a little bit faster.

**Comment by M1ha-Shvn** [comment]
Released [v1.0.2](https://github.com/carrotquest/django-clickhouse/releases/tag/v1.0.2)
